src/chunkserver/op_request.cpp

```diff
@@ -289,9 +274,12 @@ void ReadChunkRequest::Process() {
         return;
     }
 
-    /**
-     * 如果没有携带applied index，那么走raft一致性协议read
-     */
+    if (lease_status.state != braft::LEASE_NOT_READY) {
```

Comment:
It is recommended that `XXXChunkRequest` does not need to directly expose `lease_status`

if (node_->IsLeaseLeader(lease_status)) {
} else {
}